test cases/frameworks/15 llvm/meson.build

```diff
@@ -3,23 +3,30 @@ project('llvmtest', ['c', 'cpp'], default_options : ['c_std=c99'])
 method = get_option('method')
 static = get_option('link-static')
 
-if(method == 'combination')
-  d = dependency('llvm', version : static ? '>0.1' : '>=7.0', required : false, static : static)
-  if not d.found()
-    error('MESON_SKIP_TEST llvm not found or llvm version is too low')
+d = dependency('llvm', required : false, static : static)
+if not d.found()
+  error('MESON_SKIP_TEST llvm not found.')
+endif
+
+modules_to_find = [
+  'bitwriter', 'asmprinter', 'executionegine', 'mcjit', 'target',
```

Comment:
Tests fail because of this typo. Should be executionengine. Will fix later today.